modules/validations/invalid_format.js

```diff
@@ -40,31 +46,62 @@ export function validationFormatting() {
-                .split(';')
-                .map(function(s) { return s.trim(); })
-                .filter(function(x) { return !isSchemePresent(x); });
+        const websiteValidationIssueBase = {
+            type: type,
+            subtype: 'website',
+            severity: 'warning',
+            message: function(context) {
+                var entity = context.hasEntity(this.entityIds[0]);
+                return entity ? t.append('issues.invalid_format.website.message' + this.data,
+                    { feature: utilDisplayLabel(entity, context.graph()), site: this.value }) : '';
+            },
+            reference: showReferenceWebsite,
+            entityIds: [entity.id]
+        };
 
-            if (websites.length) {
-                issues.push(new validationIssue({
-                    type: type,
-                    subtype: 'website',
-                    severity: 'warning',
-                    message: function(context) {
-                        var entity = context.hasEntity(this.entityIds[0]);
-                        return entity ? t.append('issues.invalid_format.website.message' + this.data,
-                            { feature: utilDisplayLabel(entity, context.graph()), site: websites.join(', ') }) : '';
-                    },
-                    reference: showReferenceWebsite,
-                    entityIds: [entity.id],
-                    hash: websites.join(),
-                    data: (websites.length > 1) ? '_multi' : ''
-                }));
+        Object.entries(entity.tags).forEach(function([key, tag]) {
+            if (!/\b(website|url)\b/i.test(key)) return;
+            if (!tag) return;
+            var value = tag.trim();
+            if (!value) return;
+            if (value.includes(';')) {
+
+                var parts = value.split(';').map(function(s) { return s.trim(); });
+                var invalidParts = parts.filter(function(x) { return !isValidURL(x); });
+                if (invalidParts.length) {
+                    issues.push(new validationIssue({
+                        ...websiteValidationIssueBase,
+
+                        value: invalidParts.join(', '),
+
+                        hash: key + '=' + invalidParts.join(),
+                        data: (invalidParts.length > 1) ? '_multi' : ''
+                    }));
+                } else if (!isValidURL(value)) {
+                    // All split parts valid, but whole value still invalid
+                    issues.push(new validationIssue({
+                        ...websiteValidationIssueBase,
+
+                        value: value,
+
+                        hash: key + '=' + value,
+                        data: ''
+                    }));
+                }
+            } else {
+                // No semicolon, validate whole value
+                if (!isValidURL(value)) {
+                    issues.push(new validationIssue({
+                        ...websiteValidationIssueBase,
+
```

Comment:
As for https://github.com/openstreetmap/iD/issues/11436#issuecomment-3323878608:
```suggestion
                        key: key,
                        fixes: [{ protocol: 'https', icon: 'temaki-lock' }, { protocol: 'http', icon: 'iD-icon-info-filled' }]
                            .filter(f => isValidURL(...))
                            .map(f => new validationIssueFix({
                                icon: f.icon,
                                title: t('issues.fix.add_' + f.protocol + '.title'),
                                onClick: function(context) {...}
                            })),
```